src/backends/tensorrt/plan_backend.cc

```diff
@@ -3024,6 +3054,12 @@ PlanBackend::Context::Run(
         output_copy_stream_, events_[next_set_].ready_for_output_, 0);
   }
 
+  bool zero_copy_support = false;
+  FAIL_ALL_AND_RETURN_IF_ERROR(
```

Comment:
Isn't this saved in the class, we shouldn't calculated it against for every request